docs/source/admin/traffic_vault.rst

```diff
@@ -79,6 +79,41 @@ Administration of the PostgreSQL database for Traffic Vault
 
 Similar to administering the Traffic Ops database, the :ref:`admin <database-management>` tool should be used for administering the PostgreSQL Traffic Vault backend.
 
+.. program:: reencrypt
+
+app/db/reencrypt/reencrypt
+--------------------------
+The :program:`app/db/reencrypt/reencrypt` binary is used to re-encrypt all data in the Postgres Traffic Vault with a new base64-encoded AES key.
+
+.. note:: For proper resolution of configuration files, it's recommended that this binary be run from the ``app/db/reencrypt`` directory.
+
+Usage
+-----
+``./reencrypt [options]``
+
+Options and Arguments
+---------------------
```

Comment:
These headings all use the same heading level - I think what you want is for "Usage" and "Options and Arguments" to be subsections of "app/db/reencrypt/reencrypt". You might want to review the [docs' sectioning guide](https://traffic-control-cdn.readthedocs.io/en/latest/development/documentation_guidelines.html#section-headings).